src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj

```diff
@@ -17,7 +17,7 @@ Microsoft.Data.Sqlite.SqliteException
 Microsoft.Data.Sqlite.SqliteFactory
 Microsoft.Data.Sqlite.SqliteParameter
 Microsoft.Data.Sqlite.SqliteTransaction</Description>
-    <TargetFrameworks>net451;netstandard1.2</TargetFrameworks>
+    <TargetFrameworks>net46;netstandard1.2</TargetFrameworks>
```

Comment:
By cripple, I mean: without crosstargeting calling DbDataReader.Close() is a no-op on net45x